http/events.go

```diff
@@ -21,6 +21,7 @@ import (
 	"math/rand"
 	"net"
 	"net/http"
+	"net/url"
```

Comment:
Please can you revert the changes to this file, as they have been addressed elsewhere.